build.py

```
@@ -55,25 +55,335 @@ def _load_releases() -> dict[str, str]:
+                "-o",
+                str(test_bin),
+                str(src),
+            ]
+        )
+        env = {**os.environ, "LLVM_PROFILE_FILE": str(profraw)}
+        run([str(test_bin)], env=env)
+        profdata_exe = bins / f"llvm-profdata{dot_exe}"
+        run([str(profdata_exe), "merge", "-o", str(profdata), str(profraw)])
+
+    # llvm-cov report
+    result = subprocess.run(
```

Ruff (0.15.18)

[error] `subprocess` call: check for execution of untrusted input

(S603)